src/main/java/eu/dissco/core/digitalspecimenprocessor/service/FdoRecordService.java

```diff
@@ -136,6 +137,15 @@ private JsonNode genRequestAttributes(DigitalSpecimen specimen) throws PidCreati
     return attributes;
   }
 
+  private String setPhysicalIdType(DigitalSpecimen specimen) {
+    var physicalIdType = specimen.attributes().get("ods:physicalSpecimenIdType");
+    if (physicalIdType == null || physicalIdType.asText().equals("combined")) {
```

Comment:
Think it might be good to log a warning if it is null, it probably means something goes wrong somewhere